src/pages/DistributionsPage/DistributionForm.tsx

```diff
@@ -78,6 +99,19 @@ export function DistributionForm({
   const [giftSubmitting, setGiftSubmitting] = useState(false);
   const [fixedSubmitting, setFixedSubmitting] = useState(false);
 
+  const [previousLockedTokenDistribution, setPreviousLockedTokenDistribution] =
+    useState({} as any);
+
+  const loadPreviousLockedTokenDistribution = () => {
+    getPreviousLockedTokenDistribution(epoch.id).then(result =>
+      setPreviousLockedTokenDistribution(result)
+    );
+  };
+
+  useEffect(() => {
+    loadPreviousLockedTokenDistribution();
```

Comment:
this should not run unless the hedgey integration is enabled. maybe there's a way to move it into a subcomponent as I mentioned in [this comment](https://github.com/coordinape/coordinape/pull/1769#discussion_r1051263568)

Reply:
or, as an alternative, add it into the existing `getEpochData` query so that we can avoid sending a separate API request

btw, the naming is a bit misleading -- the `getPreviousDistribution` method uses "previous" in a different sense, because it's looking for a distribution from the previous _epoch_

Reply:
addressed in afe64b0f